newsletter october (#268)
maplibre/maplibre.github.io: content/news/2023-10-30-maplibre-newsletter-october-2023/index.md

---
title: "MapLibre Newsletter October 2023"
date: 2023-10-30
categories: ["newsletter"]
authors: [wipfli, harel, bart]
draft: false
---

# MapLibre Newsletter - October 2023

## MapLibre GL JS

Just in time for this newsletter, MapLibre GL JS has gained over [5000 Stars on GitHub](https://github.com/maplibre/maplibre-gl-js/stargazers). Great that we got so far and thanks to everyone who contributed in the past and made this possible!

In the last month, 5 releases of MapLibre GL JS were made. Below you find some highlights, for a full list of improvements please have a look at the [changelog](https://github.com/maplibre/maplibre-gl-js/blob/main/CHANGELOG.md).

> - Safari DEM fix: In a new version of the Safari browser, Apple has introduced some random noise in images in private mode. While usually users don't notice this effect, it had drastic consequences for terrain3d and client-side hillshading. Mike Barry fixed this issue by using a JavaScript library for decoding PNGs rather than the browser api. See [pull request](https://github.com/maplibre/maplibre-gl-js/pull/3185).

Reviewer: Not accurate, he ended up using the browser's video frame API.

Author: Fixed

> - Safari DEM fix: In a new version of the Safari browser, Apple has introduced some random noise in images in private mode. While usually users don't notice this effect, it had drastic consequences for terrain3d and client-side hillshading. Mike Barry fixed this issue by using a JavaScript library for decoding PNGs rather than the browser api. See [pull request](https://github.com/maplibre/maplibre-gl-js/pull/3185).

Author: ```suggestion
- Safari DEM fix: In a new version of the Safari browser, Apple has introduced some random noise in images in private mode. While usually users don't notice this effect, it had drastic consequences for terrain3d and client-side hillshading. Mike Barry fixed this issue by using the browser's video frame API to decode PNGs. See [pull request](https://github.com/maplibre/maplibre-gl-js/pull/3185).
```
